src/BackpackServiceProvider.php

```diff
@@ -46,7 +46,8 @@ class BackpackServiceProvider extends ServiceProvider
      */
     public function boot(Router $router)
     {
-        $this->loadViewsWithFallbacks();
+        $this->loadViewsWithFallbacks('crud');
+        $this->loadViewsWithFallbacks('ui', 'backpack.ui');
```

Comment:
Why two calls to the same function here ? It seems weird when not inside a loop. 

Why not `loadViewsWithFallbacks('crud', '['ui', 'backpack.ui']')` and make the function aceept any number of arguments, compatible ? 

```php
public function loadViewsWithFallbacks(... $dirs) {
    foreach($dirs as (array) $dir) {
        $dir = $dir[0];
        $namespace = $dir[1] ?? null;
    }
}
```

Reply:
Nah... I think moving this function from single to multiple would make both the calling less readable... and the function itself. So I don't think that would be a change for the good, sorry.